### PR TITLE
LPS-146399 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-146399

### CODE REVIEW

**Comment by weller0246** [comment]
@gaborlovasdotcom I implemented the missing informations

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase [hunk 1]
```diff
@@ -119,6 +119,39 @@ definition {
...
+		Staging.configurePublishToLive(checkPageName = "Staging Test Page 1");
+
+		Staging.publishToLive(
+			publicationName = "sumCheck",
+			remoteStaging = "true");
+
+		Staging.openStagingAdmin(siteURLKey = "site-name");
+
+		Staging.viewPublicationSummary(
+			pagesNotToPublish = "Staging Test Page 2",
```
[the remark this answers is not in the document]
Comment (weller0246) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase:150]: Valid reasons, I apply these changes and re-run the test suite!

**Comment by weller0246** [comment]
ci:test:staging:nocompile

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**